src/views/Type.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <main class="type">
+    <h1 v-if="this.error">Resource Type not found.</h1>
+
+    <h1 v-if="!this.error">{{this.resourceType.name}}</h1>
+    <div class="back-link-container">
+      <router-link to="/types">&#60; Back to Types</router-link>
+    </div>
+    <div class="preview-container">
+      <ListSection :list="resourceTypeList" />
```

Comment:
This element appears to be clickable, as it includes a link to itself.
Maybe it `ListSection` could offer a *readOnly* or *disabled* prop to be able to disable linking in some cases.
Not a problem, just a remark.

Reply:
Will fix that in the next feature branch